avail-bootstrap/src/main/resources/avail/tools/bootstrap/generated/PrimitiveNames_en.properties

```diff
@@ -1503,6 +1503,22 @@ P_DoubleModulus_comment=\
 \ * @returns "{5}"\n\
 \ *    An approximation of `0` mod '{'@param "{2}"}.\n\
 \ */\n
+# DoubleSine : _=1
+P_DoubleSine=sin_
+P_DoubleSine_1=theta
+P_DoubleSine_comment=\
+/**\n\
+\ * Compute and answer an approximation of sin '{'@param "{1}"}, i.e. the\n\
+\ * ratio of the altitude to the distance between the origin and a point projected at\n\
```

Comment:
Please move the word "at" to the next line, so that the Infallible Primitives file will be <= 80 characters wide.